src/tests/system/tests/test_sss_override.py

```diff
@@ -15,80 +15,112 @@
-    assert result is not None, "User not found by override name!"
-    assert result.uid == 999011, "User's uid does not match original value!"
-    assert result.gid == 999011, "User's gid does not match original value!"
 
+@pytest.mark.importance("high")
+@pytest.mark.topology([KnownTopology.LDAP, KnownTopology.AD, KnownTopology.Samba])
+@pytest.mark.preferred_topology(KnownTopology.LDAP)
+def test_sss_override__user_attributes(client: Client, provider: GenericProvider):
+    """
+    :title: Override username
+    :setup:
+        1. Create user and set posix attributes
+        2. Configure SSSD with "ldap_id_mapping = false" and start SSSD
+        3. Create local override and restart SSSD
+    :steps:
+        1. Lookup user and override username
+    :expectedresults:
+        1. User and override username found and values overridden
+    :customerscenario: False
+    """
+    provider.user("user1").add(
+        uid=999011, gid=999011, home="/home/user1", gecos="user", shell="/bin/bash", password="Secret123"
+    )
+    client.sssd.domain["ldap_id_mapping"] = "False"
+    client.sssd.start()
     client.sss_override.user("user1").add(name="o-user1", uid=999999, gid=888888, home="/home/o-user1")
+    client.sssd.restart()
 
     result = client.tools.getent.passwd("user1")
-    assert result is not None, "User not found!"
-    assert result.uid == 999999, "User's uid does not match override value!"
-    assert result.gid == 888888, "User's gid does not match override value!"
+    assert result is not None, "User 'user1' not found!"
+    assert result.uid == 999999, f"User's uid {result.uid} does not match override value!"
+    assert result.gid == 888888, f"User's gid {result.gid} does not match override value!"
     assert result.home == "/home/o-user1", "User's homedir does not match override value!"
 
     result = client.tools.getent.passwd("o-user1")
-    assert result is not None, "User not found by override name!"
-    assert result.uid == 999999, "Local override uid does not match override value!"
-    assert result.gid == 888888, "Local override gid does not match override value!"
+    assert result is not None, "User 'o-user1' not found by override name!"
+    assert result.uid == 999999, f"Local override uid {result.uid} does not match override value!"
+    assert result.gid == 888888, f"Local override gid {result.gid} does not match override value!"
     assert result.home == "/home/o-user1", "User's override name homedir does not match override value!"
 
 
 @pytest.mark.importance("high")
-@pytest.mark.topology(KnownTopology.LDAP)
-@pytest.mark.topology(KnownTopology.AD)
-@pytest.mark.topology(KnownTopology.Samba)
-def test_sss_overrides__overriding_group_name_and_gid(client: Client, provider: GenericProvider):
+@pytest.mark.topology([KnownTopology.LDAP, KnownTopology.AD, KnownTopology.Samba])
+@pytest.mark.preferred_topology(KnownTopology.LDAP)
+def test_sss_override__group(client: Client, provider: GenericProvider):
```

Comment:
Why is a `user` i.e. `group_member` is being overridden here, if the test is about group related attribute override?

Reply:
Existing `test_sss_overrides__overriding_group_name_and_gid` is informative compared to proposed one.